yarn-project/ethereum/src/contracts/rollup.ts

```diff
@@ -221,6 +221,16 @@ export class RollupContract {
     return this.rollup.read.getEjectionThreshold();
   }
 
+  @memoize
+  getLocalEjectionThreshold() {
+    return this.rollup.read.getLocalEjectionThreshold();
+  }
```

Comment:
We shouldn't memoize this value if it's updateable

Reply:
Think we are already do so with a few values that can be updated, but are only expected to be so rarely, can fix but will address a few more then as well:
- getProvingCostPerMana
- getManaTarget
- getManaLimit
- getLocalEjectionThreshold